lib/routes/universities/shu/jwc.js

```diff
@@ -40,23 +41,26 @@ module.exports = async (ctx) => {
             if (cache) {
                 return Promise.resolve(JSON.parse(cache));
             }
+            const date = new Date(item.date);
+            const time_zone = 8;
+            const server_offset = date.getTimezoneOffset() / 60;
             const respond = await got.get(itemUrl);
             const $ = cheerio.load(respond.data);
             const single = {
                 title: item.title,
                 link: itemUrl,
-                author: $('.normal').next().text().trim().slice(0, -3),
+                author: $('[id$=_lblUser]').text().trim(),
+                pubDate: new Date(date.getTime() - 60 * 60 * 1000 * (time_zone + server_offset)).toUTCString(),
```

Comment:
Please use timezone utility instead of reinventing the wheel.
docs: https://docs.rsshub.app/joinus/pub-date.html